frontend/src/components/maps/leaflet/InfoSlideOut/InfoContent.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { ListGroup } from 'react-bootstrap';
+import { IBuilding, IParcel, PropertyTypes } from 'actions/parcelsActions';
+import { Label } from 'components/common/Label';
+import { ParcelPIDPIN } from './ParcelPIDPIN';
+import ParcelAttributes from './ParcelAttributes';
+import './InfoSlideOut.scss';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import Claims from 'constants/claims';
+import BuildingAttributes from './BuildingAttributes';
+
+interface IInfoContent {
+  /** the selected property information */
+  propertyInfo: IParcel | IBuilding | null;
+  /** The property type [Parcel, Building] */
+  propertyTypeId: PropertyTypes | null;
+}
+
+/**
+ * Component that displays the appropriate information about the selected property
+ * in the property info slideout
+ * @param {IInfoContent} propertyInfo the selected property
+ * @param {IInfoContent} propertyTypeId the property type [Parcel, Building]
+ */
+export const InfoContent: React.FC<IInfoContent> = ({ propertyInfo, propertyTypeId }) => {
+  const keycloak = useKeycloakWrapper();
+  return (
+    <>
+      <ListGroup>
```

Comment:
This is pretty long for a screen on a small device. You may need to provide a max-length and scroll: auto in the case that not all of the details are visible on the screen.

Reply:
on it 👍